react-bricks/bricks/react-bricks-ui/singleColumnContent/Tweet/TweetLight.tsx

```
@@ -0,0 +1,261 @@
+  // const [isMouseOver, setIsMouseOver] = useState(false)
+
+  const isAdmin = false
+  const previewMode = false
+  const isMouseOver = false
+  const setIsMouseOver = (_: any) => {}
+
+  const handleClick = (tweetLink: string) => (event: React.MouseEvent) => {
+    if (isAdmin && !previewMode) {
+      return event.preventDefault()
+    }
+    if (typeof window !== undefined) {
```

Biome (2.1.2)

[error] Invalid typeof comparison.

Compare with one of the following string literals:


(lint/correctness/useValidTypeof)